internal/generate/tagstests/main.go

```diff
@@ -637,6 +649,43 @@ func (v *visitor) processFuncDecl(funcDecl *ast.FuncDecl) {
 						)
 					}
 				}
+				if attr, ok := args.Keyword["randomBgpAsn"]; ok {
```

Comment:
Should `randomBgpAsn` and `randomIPAddress` be added to the contributor docs? 
https://hashicorp.github.io/terraform-provider-aws/resource-tagging/#required-argument-parameters